src/main/java/io/supertokens/webserver/api/passwordless/ConsumeCodeAPI.java

```diff
@@ -55,6 +55,7 @@ public String getPath() {
 
     @Override
     protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOException, ServletException {
+        // API is tenant specific
```

Comment:
check everywhere for this.getTenantIdentifierWithStorageFromRequest(req).toAppIdentifierWithStorage(),